ServerCore/Pages/Teams/TeamListBase.cs

```diff
@@ -26,10 +26,10 @@ public TeamListBase(PuzzleServerContext serverContext, UserManager<IdentityUser>
 
         protected async Task LoadTeamDataAsync()
         {
-            List<Team> allTeams = await (from team in _context.Teams
-                                         where team.Event == Event
-                                         select team).ToListAsync();
-
+            List<Team> allTeams = null;
```

Comment:
super nit - could remove the part that sets to null.

Reply:
fixed :)